templates/web/brent/waste/garden/subscribe_existing_bin_desc.html

```diff
@@ -1,6 +1,6 @@
 <div class="waste-service-hint-banner">
     <div class="waste-service-hint-banner__image">
-        <img src="/i/waste-containers/bin-black-brown-lid-recycling.png" alt="">
+        <img src="/i/waste-containers/bin-green.png" alt="Green garden waste bin">
```

Comment:
The alt text being blank was okay because it's only a decorative image, the text that will be changed below can describe it if need be, as on other cobrands.